airbyte-workers/src/main/java/io/airbyte/workers/process/AsyncOrchestratorPodProcess.java

```diff
@@ -99,8 +99,10 @@ private int computeExitValue() {
 
     // trust the doc store if it's in a terminal state
     if (docStoreStatus.equals(AsyncKubePodStatus.FAILED)) {
+      log.info("State Store reports orchestrator pod {} failed", getInfo().name());
```

Comment:
Should this be `error` to stand out in the logs or is this truly an informational message about the failure?

Reply:
Good catch. This should be an error.